File: 프로그래머스/0/181867. x 사이의 개수/x 사이의 개수.js
function solution(myString) {
    let answer = [];
    let count = 0;
    for(let i = 0; i < myString.length; i++){
        if(myString[i] == 'x'){
            answer.push(count);
            count = 0;
        }
        else count++;
    }
    answer.push(count);
    return answer;
}

File: 프로그래머스/0/181867. x 사이의 개수/x 사이의 개수.java
import java.util.List;
import java.util.ArrayList;
import java.util.stream.Collectors;
class Solution {
    public int[] solution(String myString) {
        List<Integer> list = new ArrayList<>();
        int count = 0;
        for(int i = 0; i < myString.length(); i++){
            if(myString.charAt(i) == 'x'){
                list.add(count);
                count = 0;
            }
            else count++;
        }
        list.add(count);
        int[] ints = list.stream()
                         .mapToInt(Integer::intValue) 
                         .toArray();
        return ints;
    }
}